src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -249,15 +249,14 @@ struct LottieGenericProperty : LottieProperty
         return frame->interpolate(frame + 1, frameNo);
     }
 
-    T& operator=(const T& other)
```

Comment:
I couldn't see any differences between before and after. 
Could you please explain why this [operator -> shallowCopy] is necessary?

Reply:
I also see it's not necessary in current code.
Will drop the commit.